documentation/docs/14-configuration.md

```diff
@@ -132,8 +127,26 @@ Options related to [creating a package](#packaging).
 
 - `dir` - output directory
 - `emitTypes` - by default, `svelte-kit package` will automatically generate types for your package in the form of `d.ts.` files. While generating types is configurable, we believe it is best for the ecosystem quality to generate types, always. Please make sure you have a good reason when setting it to `false` (for example when you want to provide handwritten type definitions instead)
-- `exports` - contains an `includes` and an `excludes` array which specifies which files to mark as exported from the `exports` field of the `package.json`. Will merge existing values if available with values from `package.json` taking precedence
-- `files` - contains an `includes` and an `excludes` array which specifies which files to process and copy over when packaging
+- `exports` - a function with the type of `(filepath: string) => boolean`. When `true`, the filepath will be included in the `exports` field of the `package.json`. Any existing values in the `package.json` source will be merged with values from the original `exports` field taking precedence
+- `files` - a function with the type of `(filepath: string) => boolean`. When `true`, the file will be processed and copied over to the final output folder, specified in `dir`
+
+```js
+// svelte.config.js
+// example usage with a globbing library
```

Comment:
I think I missed this comment earlier and I thought this was demonstrating a default or something. I somewhat question if we need this example, but if we want to keep it, I'd probably remove this comment and instead put it in the prose above like `For advanced file matching you can use these options in conjunction with a globbing library:`